common/lib/xmodule/xmodule/assets/vertical/public/js/vertical_student_view.js

```diff
@@ -13,4 +13,25 @@ window.VerticalStudentView = function(runtime, element) {
             apiUrl: $bookmarkButtonElement.data('bookmarksApiUrl')
         });
     });
+    console.log("jcd VerticalStudentView loaded");
+    $(element).find('.vert').each(function(idx, completable) {
+        console.log(completable.dataset);
+        if (completable.dataset.completable != "True") {
+            return;
+        }
+        const block_key = completable.dataset.id;
+        console.log("jcd Element loaded " + block_key);
+        if (block_key) {
+            const data = {
+                'event_type': 'completion',
+                'block_key': block_key,
+		'completion': 1.0
```

Comment:
Nit: inconsistent indentation here. Also, you can use trailing commas on the last element in JS arrays/objects, just like in python.